salt/edx/prod.sls

```diff
@@ -13,6 +13,19 @@
 {% set theme_name = salt.pillar.get('edx:edxapp:THEME_NAME', None) -%}
 {% set theme_branch = salt.pillar.get('edx:edxapp:custom_theme:branch', 'mitx') -%}
 {% set theme_dir = salt.pillar.get('edx:edxapp:EDXAPP_COMPREHENSIVE_THEME_DIR', '/edx/app/edxapp/themes') -%}
+{% set os_packages = salt.pillar.get('edx:edxapp:os_packages:os_pkg',
```

Comment:
I would skip the last `os_pkg` in the key space unless you have a specific reason for it. You could probably even get away with `edx:os_packages` or `edx:dependencies:os_packages`